crosscat/utils/inference_utils.py

```diff
@@ -90,12 +85,12 @@ def mutual_information(M_c, X_Ls, X_Ds, Q, n_samples=1000):
         Linfoot.append(Linfoot_sample)
 
 
-    assert(len(MI) == len(Q))
-    assert(len(Linfoot) == len(Q))
+    assert len(MI) == len(Q)
+    assert len(Linfoot) == len(Q)
 
     return MI,  Linfoot
 
-def calculate_MI_bounded_discrete(X, Y, M_c, X_L, X_D):
+def calculate_MI_bounded_discrete(X, Y, M_c, X_L, _X_D):
```

Comment:
Agree with @axch.  _foo is a de facto standard across many languages to mean unused parameter.